docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Caffe.md

```diff
@@ -38,13 +38,13 @@ The following list provides the Caffe-specific parameters.
 CLI Examples Using Caffe-Specific Parameters
```

Comment:
It directly and unconditionally refers to CLI parameters. Should be replaced by Python API.

Reply:
I suppose we are not to update it, as long as it is going to be deprecated, right?